features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,700 @@
+
+    Bij het bepalen of een resource geometrie binnen de opgegeven contour ligt (bbox of polygoon) of 
+    dat een opgegeven geometrie (punt) binnen de resource geometrie ligt, wordt een intersects spatial operator gebruikt.
+
+    Intersects:
+    Om te bepalen of een geometrie A binnen een geometrie B ligt, deze raakt, kruist of (deels) overlapt, wordt gebruik gemaakt van de spatial operator INTERSECTS:
+
+      intersects(geometrie A, geometrie B)
+
+    Intersects betekent dat geometrie A en geometrie B tenminste één gemeenschappelijk punt hebben.
+    Anders gezegd geometrie A en geometrie B mogen niet volledig los van elkaar staan.
+    Het resultaat van intersects(geometrie A, geometrie B) levert hetzelfde op als intersects(geometrie B, geometrie A).
+    Het maakt niet uit of geometrie A of geometrie B de geometrie is waarmee gezocht wordt en
+    het maakt niet uit of geometrie A of geometrie B de resource geometrie is.
+    Als intersects true oplevert, dan wordt de resource waar de geometrie onderdeel van is, geretourneerd.
+
+    Referenties:
+    - Haal Centraal Common - foutafhandeling.feature
+    
+    Het geometrisch zoeken met een punt geometrie kan met de parameter: locatie.
+
+    Het geometrisch zoeken met een bounding box geometrie kan met de parameter: bbox.
+    Het is ook mogelijk om een bounding box op te geven waarbij de coördinaten van de linker onderhoek en de rechter bovenhoek exact gelijk zijn.
+    De werking is dan gelijk aan het zoeken met een locatie.
+    
+    Het geometrisch zoeken met een polygoon geometrie kan met de parameter: polygoon.
+
+
+    Ondersteund worden:
+    - het zoeken van panden op een locatie
+    - het zoeken van adresseerbare objecten binnen een bounding box
+    - het zoeken van panden binnen een bounding box
+    - het zoeken van adressen binnen een vrije contour (polygoon, multipolygoon)
+      Omdat adres een samengesteld object is en geen geometrie bevat, worden adressen bepaald op basis van het adresseerbare object dat aan een adres is gekoppeld.
+      Daarbij worden hoofd- en nevenadressen van adresseerbare objecten geleverd, waarbij de adresseerbare object geometrie volgens de intersect spatial operator matcht met de opgegeven contour.
+
+    Deze feature beschrijft geen scenario's waarbij objecten een geometrie van het type multi polygoon bevatten.
+    Het enige object type waar dit voor geldt is een woonplaats, echter wordt het zoeken van woonplaatsen met locatie, bbox of polygoon niet ondersteund.
+
+    Het zoeken met een multipolygoon wordt bij adressen ondersteund.
+    Deze scenario's zijn niet apart beschreven, omdat het zoeken met een multipolygoon overeenkomt met het meerdere malen zoeken met een polygoon waarbij het resultaat wordt ontdaan van evt. dubbelingen.
+
+    In de hierna volgende voorbeelden is een vereenvoudigde representatie gebruikt voor coördinaten.
```

Comment:
in welke zin is die vereenvoudigd? als ik het zo zie is ook in de Gegeven en in de Als (requests) geen RD (EPSG:28992) gebruikt. dat betekent dat deze scenario's niet zouden werken op de API